ansible/www-standalone/tasks/site-setup.yaml

```diff
@@ -110,11 +110,9 @@
     dest: /etc/crontab
     line: "{{ item }}"
   with_items:
-    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh nodejs'
-    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh iojs'
-    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
-    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh iojs'
-    - '*  *    * * *   root    /home/nodejs/cdn-purge.sh'
+    - '*/30 *    * * *   dist    /home/staging/tools/promote/promote_nightly.sh nodejs'
+    - '*/5  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
```

Comment:
This checks whether index.json is newer than the website's index.html, and if it is, triggers a full site rebuild which in turn triggers a cache purge. This is done so that the website has the latest release on its front page without that having to be triggered by nodejs/nodejs.org.